proposals/Template-Instantiation.md

```diff
@@ -338,7 +338,7 @@ To make this process more streamlined, we propose treating inner template elemen
 };
 ```
 
-With this interface, the template process callback that implements foreach and other kinds of looping constructs could simply call `replace` with newly constructed elements as follows:
+With this interface, the template process callback that implements a *foreach* and other kinds of looping constructs could simply be called `replace` with newly constructed elements, as follows:
```

Comment:
The original sentence was correct. "The template process callback ~ could simply call `replace` with newly constructed elements,".